layer/USAGE_android.md

```diff
@@ -1,143 +1,151 @@
  * Click on "Run" in the menu
  * Choose "Edit Configurations..."
  * In the dialog box, look for the "Install Flags:" text box
  * Enter `-g`
  * Click "Apply"
 
-If this does not work, you may still require enabling permissions for
-your application from the settings menu.
-
-**Failure to do so** will result in your application crashing during `vkCreateInstance`
-since the layer will attempt, but fail, to create the capture file.
-
-### Disabling Memory Tracking Seg-Faults
-
-The GFXReconstruct layer sometimes uses page table faults to properly
-track memory usage by an application.
-Because of this, the SIGSEGV page fault event can cause unnecessary
-breaking when stepping through a debugger while your application is
-running.
-
-To work around this, you can use the following command to stop the
-debugger from breaking on SIGSEGVs generated by the capture layer's
-memory tracking:
-
+It may also be possible to grant the external storage permission to
+an installed application through the device Settings.
+
+**Failure to enable external storage** can cause the layer to crash if
+if it tries and fails to create a capture file on the external storage
+device.
+
+### Disabling Debug Breaks Triggered by the GFXReconstruct Layer
+The Vulkan API allows Vulkan memory objects to be mapped by an application
+for direct modification. To successfully capture an application,
+the GFXReconstruct layer must be able to detect when the application
+modifies the mapped memory. The layer can be configured to detect memory
+modifications by marking the mapped memory as write protected, triggering
+an access violation when the application writes to the memory. The
+layer then uses a signal handler to intercept the signal generated by the
+access violation, where it removes the write protection, marks the modified
+memory page as dirty, and allows the application to continue.
+
+When running an application in a debugger with the layer enabled, the
+access violations triggered by the layer's memory tracking behavior may
+cause the debugger to break. These debug breaks may be disabled for LLDB with
+the following command:
 ```
 process handle SIGSEGV -n true -p true -s false
 ```
 
-This can be entered manually through the LLDB tab on Android Studio's
+This command may be entered manually through the LLDB tab on Android Studio's
 Debug panel.
 
-It can also be set as a post launch command in the project configuration:
-
-```
-Run > Edit Configurations...
-```
-
-Then choosing:
-
-```
-Debugger > LLDP Post Attach Commands
-```
-
-Where you can click the `+` and add it manually.
-
-## Globally Enabling the Layer
+It may also be set as a post launch command in the project configuration:
+ * Click on "Run" in the menu
+ * Choose "Edit Configurations..."
+ * In the dialog box, select the "Debugger" tab
+ * In the "Debugger" tab, select the "LLDB Post Attach Commands" tab
+ * Click the `+` to add the command to the command list
+ * Click "Apply"
```

Comment:
I'm not terribly familiar with Android, but isn't there a missing step here?  E.g., actually entering the `process` command given above?